docs/installation/index.md

```diff
@@ -22,15 +22,15 @@ Refer to [REP-2000](https://www.ros.org/reps/rep-2000.html) for the system depen
 
 See [here](autoware) for how to install Autoware.
 
-## Installation steps of tools for users
+## Tools for users
```

Comment:
If you change here, I think it's better to write like this.

```md
## Installation steps

### Autoware

### Tools for users
```

Reply:
Also, if there is only one word "tools", I think we can change this section as well.

```suggestion
## Related tools
```

Or this?
```suggestion
## Peripheral tools
```

@LalithVipulananthan Any ideas?

Reply:
Thank you for your review and almost agree with you.

I think shallow hierarchy is more simpler?
like
```
- Installation
  - Installation steps of Autoware
  - Installation steps of related tools
  - Additional settings for developer
```
than 
```
- Installation
  - Installation steps 
    -  Autoware
    - Related tools
  - Additional settings for developer
```

Reply:
A shallow hierarchy seems fine, but `Installation steps of Autoware` and `Installation steps of related tools` should be shortened to `Autoware installation` and `Related tools` IMO.

On a related note, I think that the Autoware installation page is so short that we could perhaps merge it with the index page? 🤔

Reply:
> A shallow hierarchy seems fine, but Installation steps of Autoware and Installation steps of related tools should be shortened to Autoware installation and Related tools IMO.

I agree!


> we could perhaps merge it with the index page? 

You mean [installation/index.md#autoware-installation](https://github.com/autowarefoundation/autoware-documentation/blob/f3b835552193a1f5c816d98741b2f00bd702ac24/docs/installation/index.md#autoware-installation) section is better to be merged with [installation/autoware/index.md](https://github.com/autowarefoundation/autoware-documentation/blob/f3b835552193a1f5c816d98741b2f00bd702ac24/docs/installation/autoware/index.md).

Yes, I agree it is simpler.